mesheryctl/internal/cli/root/system/context/context.go

```diff
@@ -48,7 +48,7 @@ var ContextCmd = &cobra.Command{
 
 func init() {
 	homeDir, err := os.UserHomeDir()
-	viper.SetConfigName("mesheryctlconfig")
```

Comment:
Hey this was the config file name

Reply:
Yeah, I figure we can simplify by calling it "config" akin to what kubectl does.